.env.stable

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-9556adb
+PFCP_AGENT_IMAGE=ghcr.io/omec-project/upf-epc/upf-epc-pfcpiface:pr-547
```

Comment:
To be changed, added to test this change with https://github.com/omec-project/upf/pull/547